tests/contracts/helloWorld/output_9.json

```diff
@@ -1,10 +1,14 @@
 {
-  "gas_remaining": "7573",
+  "gas_remaining": "7813",
   "errors": [
     {
       "error_message":
-        "Sending more than one message not currently permitted",
-      "start_location": { "file": "", "line": 0, "column": 0 },
+        "Identifier \"one_msg\" is not bound in environment:\n",
```

Comment:
@jjcnn Can you retain `one_msg` so that this error remains as before?

Reply:
Sorry, I was a bit too quick there.

Fixed.